<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Page_2</class>
 <widget class="QWidget" name="Page_2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="minimumSize">
      <size>
       <width>505</width>
       <height>0</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>527</width>
        <height>597</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" rowspan="8">
        <widget class="QFrame" name="frame_1">
         <property name="minimumSize">
          <size>
           <width>10</width>
           <height>560</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>10</width>
           <height>560</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>系统性能优化</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" rowspan="8">
        <widget class="QFrame" name="frame_2">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>560</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>560</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>AcadEref</family>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QToolButton" name="Page2_toolButton">
         <property name="minimumSize">
          <size>
           <width>360</width>
           <height>28</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <underline>false</underline>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="tabletTracking">
          <bool>false</bool>
         </property>
         <property name="focusPolicy">
          <enum>Qt::WheelFocus</enum>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::PreventContextMenu</enum>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'宋体'; font-size:10pt;&quot;&gt;Speccy就是制作CCleaner的公司Piriform所开发的第四款软件，Speccy的界面非常简单易于使用为我们提供了硬件的各项技术指标，它可以查看电脑内的软硬件信息，比如操作系统、CPU、内存、显卡、硬盘、网络等等，还可以检测出系统的安装日期。&lt;/span&gt; &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="whatsThis">
          <string/>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
         <property name="popupMode">
          <enum>QToolButton::DelayedPopup</enum>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>false</bool>
         </property>
         <property name="arrowType">
          <enum>Qt::NoArrow</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QToolButton" name="Page2_toolButton_2">
         <property name="minimumSize">
          <size>
           <width>360</width>
           <height>28</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'宋体'; font-size:10pt;&quot;&gt;CPU-Z是最权威的处理器和相关系统信息识别工具。它体积小巧，界面简洁、启动很快，可准确的检测出CPU、主板、内存、显卡、SPD等硬件信息。利用它你可见到自己的CPU是哪家厂牌、内频、Cache等玩家常提到的数据，更包括SelfSnoop CMOVccInstruction这些专家才看得懂得资讯与数据，让你更了解你的CPU。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QToolButton" name="Page2_toolButton_3">
         <property name="minimumSize">
          <size>
           <width>360</width>
           <height>28</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'宋体'; font-size:10pt;&quot;&gt;CPU-Z是最权威的处理器和相关系统信息识别工具。它体积小巧，界面简洁、启动很快，可准确的检测出CPU、主板、内存、显卡、SPD等硬件信息。利用它你可见到自己的CPU是哪家厂牌、内频、Cache等玩家常提到的数据，更包括SelfSnoop CMOVccInstruction这些专家才看得懂得资讯与数据，让你更了解你的CPU。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QToolButton" name="Page2_toolButton_4">
         <property name="minimumSize">
          <size>
           <width>360</width>
           <height>28</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'宋体'; font-size:10pt;&quot;&gt;GPU-Z是一款显卡识别工具，甚至可以鉴别假卡（NVIDIA）。GPU-Z是一款轻量级显卡测试软件。绿色免安装，界面直观，运行后即可显示GPU核心，以及运行频率、带宽等，如同CPU-Z一样，这也是款必备工具。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QToolButton" name="Page2_toolButton_5">
         <property name="minimumSize">
          <size>
           <width>360</width>
           <height>28</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'宋体'; font-size:10pt;&quot;&gt;硬盘里的数据至关重要，因此时常检查硬盘健康状况是有必要的。DiskInfo硬盘检测工具通过读取S.M.A.R.T了解硬盘健康状况。打开它，您就可以迅速读到本机硬盘的详细信息，包括接口、转速、温度、使用时间等DiskInfo还会根据S.M.A.R.T的评分做出评估，当硬盘快要损坏时还会发出警报，支持简体中文。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QToolButton" name="Page2_toolButton_6">
         <property name="minimumSize">
          <size>
           <width>360</width>
           <height>28</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'宋体'; font-size:10pt;&quot;&gt;BatteryMon是一款电脑的电池管理软件，可检测电池的出厂状态、电池容量及最长待机等信息，是一款绿色汉化的软件，能以图形化的方式，让我们直观的看到电池的各种性能参数。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QToolButton" name="Page2_toolButton_7">
         <property name="minimumSize">
          <size>
           <width>360</width>
           <height>28</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'宋体'; font-size:10pt;&quot;&gt;Speccy就是制作CCleaner的公司Piriform所开发的第四款软件，Speccy的界面非常简单易于使用为我们提供了硬件的各项技术指标，它可以查看电脑内的软硬件信息，比如操作系统、CPU、内存、显卡、硬盘、网络等等，还可以检测出系统的安装日期。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
